index.js

```diff
@@ -48,9 +48,13 @@ Guard.prototype = {
       }
 
       if (!Array.isArray(permissions)) {
-        return next(new UnauthorizedError('permissions_invalid', {
-          message: 'Permissions should be an Array. Bad format?'
-        }))
+        if (typeof permissions === 'string') {
```

Comment:
Agreed, that would be the more intuitive way to write it. Did it this way to maintain the same code path for existing Array usage. Happy to refactor.